README.md

```diff
@@ -1 +1,158 @@
-# portfolio
\ No newline at end of file
+<a name="readme-top"></a>
+
+
+
+<div align="center">
+  <h1><b>README File</b></h3>
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+
+# 📖 Rodolfo Carrillo Portfolio <a name="about-project"></a>
+
+
+
+**Portfolio** is the first personal portfolio developed by Rodolfo Carrillo Garcia as a Microverse Student
+
+## 🛠 Built With <a name="built-with"></a>
+**HTML | CSS | Javascript**
+### Tech Stack <a name="tech-stack"></a>
+
+> - HTML, Git, CSS
+
+<details>
+  <summary>HTML</summary>
+  <ul>
+    <li><a href="https://developer.mozilla.org/es/docs/Web/HTML">Definition</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>CSS</summary>
+  <ul>
+    <li><a href="https://es.wikipedia.org/wiki/CSS">Definition</a></li>
+  </ul>
+</details>
+
+
+
+### Key Features <a name="key-features"></a>
+
+- **Add linters to repository**
+- **Use of GitFlow**
+- **Add README file**
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+- Visual Studio Code or IDE
+- Git
+- Github account
+- NodeJS
+
+
+### Setup
+
+Clone this repository to your desired folder
```

Comment:
- [OPTIONAL] I would advise you to add the link to be used to clone so that the visitor won't get confused about what you are saying.